lib/mayaUsd/CMakeLists.txt

```diff
@@ -52,6 +51,13 @@ target_compile_definitions(${PROJECT_NAME}
         $<$<BOOL:${IS_LINUX}>:GLX_GLXEXT_PROTOTYPES>
 )
 
+if(DEFINED UFE_PREVIEW_VERSION_NUM)
+    target_compile_definitions(${PROJECT_NAME}
+        PUBLIC
+            UFE_PREVIEW_VERSION_NUM=${UFE_PREVIEW_VERSION_NUM}
+    )
+endif()
```

Comment:
Fix 2019 build issue on Windows ( Fatal Error C1017 invalid integer constant expression ). 

`UFE_PREVIEW_VERSION_NUM` is a new variable that was added recently and only defined when building against UFE `0.x.x`.